catalogue/webapp/pages/item.js

```diff
@@ -1,13 +1,15 @@
 // @flow
 import { type Context } from 'next';
-import Router from 'next/router';
+import NextLink from 'next/link';
 import fetch from 'isomorphic-unfetch';
 import { type IIIFManifest } from '@weco/common/model/iiif';
 import { iiifImageTemplate } from '@weco/common/utils/convert-image-uri';
 import { itemUrl } from '@weco/common/services/catalogue/urls';
 import Layout12 from '@weco/common/views/components/Layout12/Layout12';
 import PageLayout from '@weco/common/views/components/PageLayout/PageLayout';
-import Paginator from '@weco/common/views/components/Paginator/Paginator';
+import RenderlessPaginator from '@weco/common/views/components/RenderlessPaginator/RenderlessPaginator';
```

Comment:
Is this a common naming convention?

Reply:
Dunno – think I picked it up from a Vue course I took, so probably not. I guess it's an `HOC` of sorts – would we keep stuff like this in the `hoc` directory and call it e.g. `withPaginator`?

Reply:
Different thing though, HoCs, Faccs, and render props are three different ways of doing the same thing.

As mentioned, maybe a `usePagination` hook?

Reply:
Went with a render prop as discussed (for now). Will update the original Paginator to use this in a different PR.